docs/admin/install.rst

```diff
@@ -291,7 +312,7 @@ With the key and configuration available in ``dom0``, you're ready to set up Sec
 
     [securedrop-workstation-temporary]
     enabled=1
-    baseurl=https://yum.securedrop.org/workstation/dom0/f25
+    baseurl=https://yum.securedrop.org/workstation/dom0/f32
     name=SecureDrop Workstation Qubes initial install bootstrap
 
 - Download the SecureDrop Workstation config package to the curent working directory with the command:
```

Comment:
One difference here is that I get prompted to accept the import of the Qubes 4 release key:
```
dnf download securedrop-workstation-dom0-config
Fedora 36 - x86_64                              3.3 MB/s |  81 MB     00:24    
Fedora 36 openh264 (From Cisco) - x86_64        1.8 kB/s | 2.5 kB     00:01    
Fedora 36 - x86_64 - Updates                    3.3 MB/s |  21 MB     00:06    
Qubes OS Repository for VM (updates)            1.2 kB/s | 833  B     00:00    
Qubes OS Repository for VM (updates)            2.3 MB/s | 2.3 kB     00:00    
Importing GPG key 0x9E2795E9:
 Userid     : "Qubes OS Release 4 Signing Key"
 Fingerprint: 5817 A43B 283D E5A9 181A 522E 1848 792F 9E27 95E9
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-qubes-4-primary
Is this ok [y/N]: y
Qubes OS Repository for VM (updates)             60 kB/s | 137 kB     00:02    
SecureDrop Workstation Qubes initial install bo 436  B/s | 2.4 kB     00:05    
securedrop-workstation-dom0-config-0.7.0-1.fc32 136 kB/s | 125 kB     00:00    
```
seems like the right key, but it is an extra prompt.

Reply:
Made note of the prompt in 69ec8ebfa4fd1f8b9cd43c7705ab3f3321c6be7f